static.config.js

```diff
@@ -203,9 +204,9 @@ const getServicePageData = async (
     offeredBy: getOfferedByFromDepartments(servicePageData.departments),
   };
 
-  if (pagesOfGuides && pagesOfGuides[id]) {
+  if (pagesOfGuides && pagesOfGuides[servicePageData.id]) {
     // We're checking if this id is part of guide page because it may not be published and draw an error.
-    service.pageIsPartOf = pagesOfGuides[id];
+    servicePageData.pageIsPartOf = pagesOfGuides[servicePageData.id];
```

Comment:
In this a new part of v3? Querying guide pages first and then checking if service/info pages are part of them? I think I remember you saying that we used to query all guide pages for every service/info pages. Or something like that.

Reply:
this is in v2 rn https://github.com/cityofaustin/janis/blob/992901d69f3acdc4bfb66dfbc7ff01f46082ead2/static.config.js#L756